packages/next/amp.js

```diff
@@ -1 +1,2 @@
 module.exports = require('./dist/next-server/lib/amp')
+exports.__esModule = true;
```

Comment:
Even though the files being require()d here will have their own `exports.__esModule = true`, so this code is redundant from a runtime perspective, Rollup relies on static analysis, ahead of time, so it needs to be inside these files to be recognized.